Computing dependencies
Analyzing 91 sources with 0 local dependencies
ninja: Entering directory `output/docker__compose'
[1/91] check compose.__init__
[2/91] check tests.unit.__init__
[3/91] check compose.metrics.__init__
[4/91] check tests.acceptance.__init__
[5/91] check compose.cli.__init__
[6/91] check tests.unit.cli.__init__
[7/91] check compose.errors
[8/91] check compose.config.errors
[9/91] check tests.unit.metrics.__init__
[10/91] check tests.integration.__init__
[11/91] check tests.unit.config.__init__
[12/91] check compose.timeparse
[13/91] check const
[14/91] check compose.version
[15/91] check check_help
[16/91] check compose.cli.docopt_command
FAILED: /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/pyi/compose/cli/docopt_command.pyi 
/mnt/c/Users/nitram/PycharmProjects/pytype_runner/env/bin/python3 -m pytype.single --imports_info /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/imports/compose.cli.docopt_command.imports --module-name compose.cli.docopt_command -V 3.8 -o /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/pyi/compose/cli/docopt_command.pyi --analyze-annotated --nofail --quick /mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/compose/cli/docopt_command.py
File "/mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/compose/cli/docopt_command.py", line 4, in <module>: Can't find module 'docopt.DocoptExit'. [import-error]

For more details, see https://google.github.io/pytype/errors.html#import-error
[17/91] check compose.config.sort_services
[18/91] check tests.__init__
[19/91] check compose.cli.verbose_proxy
[20/91] check tests.unit.timeparse_test
[21/91] check setup
FAILED: /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/pyi/setup.pyi 
/mnt/c/Users/nitram/PycharmProjects/pytype_runner/env/bin/python3 -m pytype.single --imports_info /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/imports/setup.imports --module-name setup -V 3.8 -o /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/pyi/setup.pyi --analyze-annotated --nofail --quick /mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/setup.py
File "/mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/setup.py", line 62, in <module>: No attribute 'evaluate_marker' on module 'pkg_resources' [module-attr]

For more details, see https://google.github.io/pytype/errors.html#module-attr
[22/91] check compose.utils
FAILED: /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/pyi/compose/utils.pyi 
/mnt/c/Users/nitram/PycharmProjects/pytype_runner/env/bin/python3 -m pytype.single --imports_info /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/imports/compose.utils.imports --module-name compose.utils -V 3.8 -o /mnt/c/Users/nitram/PycharmProjects/pytype_runner/output/docker__compose/pyi/compose/utils.pyi --analyze-annotated --nofail --quick /mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/compose/utils.py
File "/mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/compose/utils.py", line 7, in <module>: Can't find module 'docker.errors'. [import-error]
File "/mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/compose/utils.py", line 8, in <module>: Can't find module 'docker.utils'. [import-error]
File "/mnt/c/Users/nitram/PycharmProjects/git_cloner/all_repos/docker__compose/compose/utils.py", line 78, in json_splitter: No attribute 'WHITESPACE' on module 'json.decoder' [module-attr]

For more details, see https://google.github.io/pytype/errors.html
[23/91] check versions
[24/91] check utils
[25/91] check compose.const
ninja: build stopped: subcommand failed.
Leaving directory 'output/docker__compose'
